examples/fastify-ssr/README.md

```diff
@@ -0,0 +1,22 @@
+# GraphQL Hooks Fastify SSR Example
+
+This example uses [Fastify](https://github.com/fastify/fastify) to serve a graphql server and do server side rendering. It's very basic but showcases the different functionality that `graphql-hooks` offers. This example is intended more for local development of the `graphql-hooks` packages.
+
+## How to use
+
+Download the example:
+
+```bash
+curl https://codeload.github.com/nearform/graphql-hooks/tar.gz/master | tar -xz --strip=2 graphql-hooks-master/examples/fastify-ssr
```

Comment:
Worth mentioning that it can just be run from the examples/ dir if you already have `graphql-hooks` locally?

We should update `CONTRIBUTING.md` to these steps in a getting started section as @jh3y mentioned in #69 too. Not a blocker for this PR, can be followed up.